hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieFileIndex.scala

```diff
@@ -251,8 +252,8 @@ case class HoodieFileIndex(spark: SparkSession,
 
   override def sizeInBytes: Long = getTotalCachedFilesSize
 
-  private def isDataSkippingEnabled: Boolean = HoodieSparkConfUtils.getBooleanConfigValue(
-    options, spark.sessionState.conf, DataSourceReadOptions.ENABLE_DATA_SKIPPING.key(), false)
+  private def isDataSkippingEnabled: Boolean = getConfigValue(options, spark.sessionState.conf,
+    DataSourceReadOptions.ENABLE_DATA_SKIPPING.key(), "false").toBoolean
```

Comment:
shouldn't we do DataSourceReadOptions.ENABLE_DATA_SKIPPING.defaultValue() instead of hard coding "false"